ics/repeatable.py

```diff
@@ -0,0 +1,273 @@
+from arrow import Arrow
+from dateutil import relativedelta
+from .utils import arrow_to_iso, iso_to_arrow
+
+
+class Repeatable:
+    """Represents a recurrence rule."""
+
+    _TYPE = "RRULE"
+    _FREQ_POSSIBLES = {"DAILY": "days",
+                       "WEEKLY": "weeks",
+                       "MONTHLY": "months",
+                       "YEARLY": "years"}
+
+    _WEEKDAY_POSSIBLES = {"SU": "Sunday",
+                          "MO": "Monday",
+                          "TU": "Tuesday",
+                          "WE": "Wednesday",
+                          "TH": "Thursday",
+                          "FR": "Friday",
+                          "SA": "Saturday"}
+
+    def __init__(self,
+                 freq="DAILY",
+                 until=None,
+                 count=None,
+                 interval=1,
+                 byday=[],
+                 bymonthday=[],
+                 bymonth=[]):
+        """Instantiate a new Repeatable rule.
+
+        :param freq: str
```

Comment:
Could we use a `datetime.timedelta` to derive the frequency, rather than a hardcoded string? E.g. `timedelta(weeks=1)` rather than `"WEEKLY"`

Reply:
The `timedelta` could be seralized later into an RFC compatible frequency